src/dash-playlist-loader.js

```diff
@@ -325,7 +325,10 @@ export default class DashPlaylistLoader extends EventTarget {
 
     // live playlist staleness timeout
     this.on('mediaupdatetimeout', () => {
-      this.refreshMedia_(this.media().id);
+      // We handle live content steering in the playlist controller
+      if (!this.media().attributes.serviceLocation) {
+        this.refreshMedia_(this.media().id);
```

Comment:
Should we check that `this.media()` exists?

Reply:
I believe `this.media()` should always return something valid here, since in all of the places `mediaupdatetimeout` is triggered, the media is set. It definitely wouldn't hurt to have an extra check, but It looks like there was not a check previously either.